Gruntfile.js

```diff
@@ -49,67 +77,49 @@ module.exports = function Gruntfile( grunt ) {
 					banner: grunt.file.read( 'build/header.browserextension.txt' )
 				},
 				files: {
-					'extension/generated.whowrotethat.css': 'src/less/index.less'
+					'dist/extension/generated.whowrotethat.css': 'src/less/index.less'
 				}
-			}
-		},
-		concat: {
-			browserextension: {
+			},
+			gadget: {
 				options: {
-					banner: grunt.file.read( 'build/header.browserextension.txt' ),
-					// Remove wrapping IIFE ( function () {}() );\n
-					process: function ( src, filepath ) {
-						// Only remove the end if we're removing the starting
-						// (function () { ... wrapper
-						if ( new RegExp( /^\( function \(\) {/ ).test( src ) ) {
-							src = src
-								.replace( /^\( function \(\) {/, '' ) // Beginning of file
-								.replace( /}\(\) \);\n$/, '' );
-						}
-						// eslint-disable-next-line quotes
-						return '/* >> Starting source: ' + filepath + " << */\n" +
-							src +
-							'/* >> End source: ' + filepath + ' << */';
-					}
+					banner: grunt.file.read( 'build/header.gadget.txt' )
```

Comment:
Should this header also be added to the generated JS file?

Reply:
So, I thought about that, but the generated JS is inserted *into* the template.

I was thinking of putting it up at the top of the template, but I need to look into how to make it replace the variables if I do that, so I figured we can do that as maybe a followup.

Reply:
Sure, let's leave it for after. This is all working in the important ways.